tests/driver_lis3mdl/main.c

```diff
@@ -0,0 +1,90 @@
+
+/*
+ * Copyright (C) 2015 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the LIS3MDL 3-axis magnetometer
+ *
+ * @author      René Herthel <rene-herthel@outlook.de>
+ *
+ * @}
+ */
+#ifndef TEST_LIS3MDL_I2C
+#error "TEST_LIS3MDL_I2C not defined"
+#endif
+#ifndef TEST_LIS3MDL_MAG_ADDR
+#error "TEST_LIS3MDL_MAG_ADDR not defined"
+#endif
+#ifndef TEST_LIS3MDL_MAG_PIN
+#error "TEST_LIS3MDL_MAG_PIN not defined"
+#endif
+
+#include <stdio.h>
+
+#include "vtimer.h"
+#include "lis3mdl.h"
+
+#define SLEEP       (100 * 1000U)
+#define MAG_ODR     LIS3MDL_ODR_10Hz
+#define MAG_SCALE   LIS3MDL_SCALE_4G
+#define MAG_XY_MODE LIS3MDL_XY_MODE_MEDIUM
+#define MAG_Z_MODE  LIS3MDL_Z_MODE_MEDIUM
+
+int main(void)
+{
+    lis3mdl_t dev;
+    lis3mdl_3d_data_t mag_value;
+    int16_t temp_value;
+
+    puts("\nLIS3MDL test application\n");
+    printf("Initializing LIS3MDL sensor at I2C_%i ... ", TEST_LIS3MDL_I2C);
+
+    if (lis3mdl_init(dev, TEST_LIS3MDL_I2C, TEST_LIS3MDL_MAG_PIN,
+                     TEST_LIS3MDL_MAG_ADDR, LIS3MDL_XY_MODE_MEDIUM,
+                     LIS3MDL_Z_MODE_MEDIUM, LIS3MDL_ODR_10Hz,
+                     LIS3MDL_SCALE_4G) == 0)
+    {
+        puts("[ OK ]\n");
+    }
+    else
+    {
+        puts("[ FAIL ]\n");
+        return 1;
+    }
+
+    while(1)
+    {
+        if (lis3mdl_read_mag(&dev, &mag_value) == 0)
+        {
```

Comment:
Please check the https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions - you could also check what astyle does for you.

This particular case:
```c
if (...)
{
```
should become
```c
if (...) {
```